google/ggrc-core
# [5OBJ] GGRC-8528 Move Contracts and Policies out of “directives” table

# Dependencies

_**This PR should be merged into [GGRC-7200](https://github.com/google/ggrc-core/tree/GGRC-7200) branch which is the common one for all 5OBJ. PRs**_

# Issue description

Contracts and Policies should be moved out of the “directives” table into the separate ones ("contracts" and "policies" respectively).

# Steps to test the changes

1. Run the migration and make sure that the Contracts & Policies data were successfully migrated into "contracts" and "policies" tables respectively, and yet the old data remained in "directives" table.

# Solution description

* Create separate DB tables for Contracts (_"contracts"_) & Policies (_"policies"_) and copy the data from _"directives"_ table there. New tables have the reduced set of columns (see technical documentation).

# Sanity checklist


# Migration checklist

# PR Review checklist
